README.md

```diff
@@ -2,26 +2,24 @@
-
-1. You don't need to set `XPRO_BASE_URL` mentioned in above link.
-2. While following steps in [Apply mitxpro-theme](https://github.com/mitodl/mitxpro-theme/blob/master/README.md#apply-mitxpro-theme) please use
-`mitx-theme` instead of `mitxpro-theme`.
-3. In [Compiling a theme](https://github.com/mitodl/mitxpro-theme/blob/master/README.md#compiling-a-theme) you need to use `mitx-theme` e.g.
-    ```
-    $ make lms-shell
-    $ paver update_assets --themes mitx-theme
-    ```
+| branch                                                         | description                                                                                     |
+|----------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| [dogwood](https://github.com/mitodl/mitx-theme/tree/dogwood)   | OBSOLETE theme for use with the `mitx-dogwood` release of Residential MITx                      |
+| [ficus](https://github.com/mitodl/mitx-theme/tree/ficus)       | OBSOLETE theme for use with the `mitx-ficus` release of Residential MITx                        |
+| [ginko](https://github.com/mitodl/mitx-theme/tree/ginko)       | OBSOLETE theme for use with the `mitx-ginko` release of Residential MITx                        |
+| [hawthorn](https://github.com/mitodl/mitx-theme/tree/hawthorn) | OBSOLETE theme for use with the `mitx-hawthorn` release of Residential MITx                     |
+| [ironwood](https://github.com/mitodl/mitx-theme/tree/ironwood) | OBSOLETE theme for use with the `mitx-ironwood` release of Residential MITx                     |
+| [koa](https://github.com/mitodl/mitx-theme/tree/koa)           | OBSOLETE theme for use with the `mitx-koa` release of Residential MITx                          |
+| [lilac](https://github.com/mitodl/mitx-theme/tree/lilac)       | OBSOLETE theme for use with the `mitx-lilac` release of Residential MITx                        |
+| [maple](https://github.com/mitodl/mitx-theme/tree/maple)       | OBSOLETE theme for use with the `mitx-maple` release of Residential MITx                        |
+| [nutmeg](https://github.com/mitodl/mitx-theme/tree/nutmeg)     | OBSOLETE theme for use with the `mitx-nutmeg` release of Residential MITx                       |
+| [olive](https://github.com/mitodl/mitx-theme/tree/olive)       | OBSOLETE theme for use with the `mitx-olive` release of Residential MITx                        |
+| [palm](https://github.com/mitodl/mitx-theme/tree/palm)         | OBSOLETE theme for use with the `mitx-palm` release of Residential MITx                         |
+| [quince](https://github.com/mitodl/mitx-theme/tree/quince)     | OBSOLETE theme for use with the `mitx-quince` release of Residential MITx                       |
+| [redwood](https://github.com/mitodl/mitx-theme/tree/redwood)   | OBSOLETE theme for use with the `mitx-redwood` release of Residential MITx                      |
```

Comment:
@arslanashraf7 Do we need to keep all these in Readme? We are using Sumac and latest.